web/templates/admin/job/form.html.eex

```diff
@@ -1,68 +1,52 @@
-<%= form_for @changeset, @action, fn f -> %>
+<%= form_for @changeset, @action, fn form -> %>
   <%= if @changeset.action do %>
     <div class="alert alert-danger">
       <p>Oops, something went wrong! Please check the errors below.</p>
     </div>
   <% end %>
 
   <div class="form-group">
-    <%= label f, :title, class: "control-label" %>
-    <%= text_input f, :title, class: "form-control" %>
-    <%= error_tag f, :title %>
+    <%= input form, :title %>
   </div>
 
   <div class="form-group">
-    <%= label f, :company, class: "control-label" %>
-    <%= text_input f, :company, class: "form-control" %>
-    <%= error_tag f, :company %>
+    <%= input form, :company %>
   </div>
 
   <div class="form-group">
-    <%= label f, :contact, class: "control-label" %>
-    <%= text_input f, :contact, class: "form-control" %>
-    <%= error_tag f, :contact %>
+    <%= input form, :contact %>
   </div>
 
   <div class="form-group">
-    <%= label f, :city, class: "control-label" %>
-    <%= text_input f, :city, class: "form-control" %>
-    <%= error_tag f, :city %>
+    <%= input form, :city %>
   </div>
 
   <div class="form-group">
-    <%= label f, :company_url, class: "control-label" %>
-    <%= text_input f, :company_url, class: "form-control" %>
-    <%= error_tag f, :company_url %>
+    <%= input form, :company_url %>
```

Comment:
Awesome! Looks good then!

> On Aug 2, 2016, at 2:09 PM, Jason Draper <notifications@github.com> wrote:
> 
> In web/templates/admin/job/form.html.eex <https://github.com/thoughtbot/community/pull/78#discussion_r73206875>:
> 
> >    </div>
> >  
> >    <div class="form-group">
> > -    <%= label f, :company_url, class: "control-label" %>
> > -    <%= text_input f, :company_url, class: "form-control" %>
> > -    <%= error_tag f, :company_url %>
> > +    <%= input form, :company_url %>
> It didn't need it to begin with.
> 
> That came from the default scaffold b/c I wanted quick views.
> 
> Good looking out!
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub <https://github.com/thoughtbot/community/pull/78/files/7c3b0e66731660559bf1fb239764a362de78b8b9#r73206875>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AABXep9Nw_DkDUSxx-0Hj5445QWGvzNRks5qb4ftgaJpZM4Ja4zK>.
>